### PR TITLE
WPF-60751 Prepare UG for Highlighting first matched item in SfTextBoxExt

### PR DESCRIPTION
**Pull request by jeya-kasipandi**
Added UG content for Highlighting first matched item in SfTextBoxExt.

### CODE REVIEW

File: wpf/Autocomplete/Dropdown-customization.md [hunk 1]
```diff
@@ -238,4 +238,49 @@ textBoxExt.SelectionBackgroundColor = new SolidColorBrush(Colors.Red);
 
 ![Selected item background color](Dropdown_customization_images/SelectionBackgroundColor.png)
 
+## Highlighting matched text in DropDown
+
+`SfTextBoxExt` supports highlighting the first matched text when a dropdown is opened by setting the `AutoHighlightMatchedItem` API to true. By setting `AutoHighlightMatchedItem` to false, you can restrict the highlighting of first matched text in `SfTextBoxExt`.
```
Comment (laksmipriyamurugesan) [inline wpf/Autocomplete/Dropdown-customization.md:243]: By setting `AutoHighlightMatchedItem` to false, you can restrict the highlighting of the first matched text in `SfTextBoxExt.`

File: wpf/Autocomplete/Dropdown-customization.md [hunk 2]
```diff
@@ -238,4 +238,48 @@ textBoxExt.SelectionBackgroundColor = new SolidColorBrush(Colors.Red);
 
 ![Selected item background color](Dropdown_customization_images/SelectionBackgroundColor.png)
 
+## Highlighting matched text in DropDown
+
+`SfTextBoxExt` supports highlighting the first matched text when a dropdown is opened by setting the `AutoHighlightMatchedItem` API to true. By setting `AutoHighlightMatchedItem` to false, you can restrict the highlighting of the first matched text in `SfTextBoxExt`.
```
Comment (Choza-rajan) [inline wpf/Autocomplete/Dropdown-customization.md:243]: `SfTextBoxExt` supports highlighting the first matched item when a dropdown is opened by setting the `AutoHighlightMatchedItem` API to true. The default value is false.

**Review by Choza-rajan — APPROVED** [review]
Fine

**Review by PaulAnderson02 — APPROVED** [review]
Fine

**Pull request merged**